docs/conf.py

```diff
@@ -10,9 +10,69 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../tdclient'))
```

Comment:
@aodag Not sure if it is a mandatory or not for autodoc. Could you give me an advice?

Reply:
if we can't install tdclient on building docs, this workaround is needed including to PYTHONPATH.

Reply:
Thanks, removed e59d7a5